sql/core/src/test/resources/sql-tests/results/regexp-functions.sql.out

```diff
@@ -334,9 +334,9 @@ org.apache.spark.SparkRuntimeException
   "errorClass" : "INVALID_PARAMETER_VALUE",
   "sqlState" : "22023",
   "messageParameters" : {
-    "expected" : "'], ['",
     "functionName" : "`regexp_extract_all`",
-    "parameter" : "regexp"
```

Comment:
But call `expected` in this scene seems unreasonable.
The value of parameter(s) '<parameter>' in <functionName> is invalid: <expected>